src/vs/base/browser/ui/button/button.css

```diff
@@ -46,6 +46,11 @@
 	padding: 4px 0;
 }
 
+.monaco-button-dropdown .monaco-button-dropdown-separator.disabled {
+	opacity: 0.4 !important;
+	cursor: default;
```

Comment:
Should the cursor style apply to the entire `.monaco-button-dropdown`?

Reply:
👍🏻